### PR TITLE
Sync demo.eclipse-pass.org marketing site with eclipse-pass.org

### PR DESCRIPTION
**Pull request by aforward**
Screen Shot 2022-12-12 at 14 28 36

Here are the instructions on how to sync the marketing site in the future.
https://github.com/eclipse-pass/main/pull/407

### CODE REVIEW

File: demo.html [hunk 1]
```diff
@@ -97,7 +97,7 @@ <h1 class="font-weight-light mt-5">Eclipse PASS Demo</h1>
             <p class="mt-5 mb-0">
               You can find instructions on how to use the demo instance <a href="https://github.com/eclipse-pass/main/blob/main/docs/user/README.md">here</a>. Interested in running pass at your institution? <a href="contact.html">Contact us!</a>
```
Comment (grant-mcs) [inline demo.html]: Not part of this change, but should `pass` here instead be `PASS`?

Comment (aforward) [reply demo.html]: Possibly yes.

**Review by grant-mcs — APPROVED** [review]
